lib/activerecord-import/adapters/sqlite3_adapter.rb

```diff
@@ -33,8 +33,8 @@ def insert_many(sql, values, *args) # :nodoc:
     value_sets.each do |value_set|
       number_of_inserts += 1
       sql2insert = base_sql + value_set.join( ',' ) + post_sql
-      first_insert_id = insert( sql2insert, *args )
-      last_insert_id = first_insert_id + value_set.size - 1
+      last_insert_id = insert( sql2insert, *args )
+      first_insert_id = last_insert_id - value_set.size + 1
```

Comment:
It looks like `insert` returns last inserted id, not first:
https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb#L126